### DISCUSSION TITLE
Batter mdYou-ed now playing screen

### DISCUSSION THREAD

**Discussion by exkpwn** [PaulWoitaschek/Voice#1336] [Ideas]
Yes

**Comment by PaulWoitaschek** [comment]
Could you provide some more context what you mean with this feature idea?

**Reply by Impostu** [reply]
> Could you provide some more context what you mean with this feature idea?

I think he meant "Better Material You style now playing screen"

**Comment by exkpwn** [comment]
![IMG_20220712_160630_132](https://user-images.githubusercontent.com/44256630/178471468-3ab4c744-16f1-478e-bbab-eb9cd914058d.jpg)
Something like that i found on telegram
